### DISCUSSION TITLE
Project highlights: community help options

### DISCUSSION THREAD

**Discussion by greatvovan** [grafana/grafana#55639] [General]
The document https://github.com/grafana/grafana/blob/main/SUPPORT.md suggests https://community.grafana.com/ as the option for getting help from community, however we also have this Discussions section on GitHub. What is the vision of the project maintainers in this aspect? Looking at the list of categories (or tags in the forum), I see lot of intersection. Having two options seems reducing efficiency of community support due to spreading incoming requests into more than one stream to monitor.

Can we probably either

1. Shut down one of the options (make it read-only as there is a lot of useful content). Or
2. If we'd like to keep both, outline the recommendations which is for what. E.g. questions of type A, B, C --> go to https://community.grafana.com/, type X, Y, Z --> go to https://github.com/grafana/grafana/discussions

?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by greatvovan** [reply]
That's great news, but my question was more about how to get community help, and whether Discussions is an appropriate stream for that.

**Reply by gelicia** [reply]
Ah, yeah I can actually answer this one a bit more directly. We did find that having discussions along with the forum and issue tickets got confusing. This is a decision we made a few years ago that did not evolve in the way we were expecting. We are closing all our discussions, and are hoping that folks make tickets on the issues of the repo for anything they would want to see. Discussions where we want to get community feedback would be served better in the forum. 

I hope this is more clear.

**Reply by greatvovan** [reply]
Thanks, that helps.
